nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const router = express.Router();
+
+const reservations = require('./../data/reservations.json');
+
+// /api/reservations
+
+router.get('/', async (request, response) => {
+    try {
+        const allReservations = reservations.map((r) => r);
```

Comment:
Since your reservations array is already up there, you can just send that and not map it before sending it.